### PR TITLE
go vet fixes

### PR DESCRIPTION
**Pull request by alexgblr**

### CODE REVIEW

**Comment by alexgblr** [comment]
https://www.pivotaltracker.com/story/show/162833172

File: controllers/authControllers.go [hunk 1]
```diff
@@ -74,19 +75,22 @@ var sendCodePerEmail = func(email string, code string) {
...
+	contentVar := gochimp.Var{
+		Name:    "main",
+		Content: "<h1>Welcome aboard!</h1>",
+	}
 	content := []gochimp.Var{contentVar}
 
 	_, err = mandrillApi.TemplateAdd(templateName, fmt.Sprintf("%s", contentVar.Content), true)
 	if err != nil {
-		fmt.Println("Error adding template: %v", err)
+		fmt.Printf("Error adding template: %v\n", err)
```
Comment (mauricext4fs) [inline controllers/authControllers.go:86]: ?? Why not Println?

**Review by mauricext4fs — CHANGES_REQUESTED** [review]
I don't even see the reason for this pull request at all.

File: controllers/authControllers.go (hunk 1 above)
Comment (alexgblr) [reply controllers/authControllers.go:86]: try running "make test". 
go vet will issue those errors.
Println is not intended to have a format specifier, printF does.

https://golang.org/pkg/fmt/#Println
Println formats **using the default formats for its operands** and writes to standard output. Spaces are always added between operands and a newline is appended. It returns the number of bytes written and any write error encountered.

https://golang.org/pkg/fmt/#Printf
Printf formats according to a **format specifier** and writes to standard output. It returns the number of bytes written and any write error encountered

There are 3 ways to do logging w/o adding redundant '\n'
1. use log package. log.Printf inserts a newline automatically
2. use fmt.Println("Error adding template: " + err.Error()) - but this quickly gets out of control when you insert something in the middle
3. use fmt.Sprintf as an argument to fmt.Printl - also ugly

**Comment by alexgblr** [comment]
https://www.pivotaltracker.com/story/show/162862349

**Comment by mauricext4fs** [comment]
ok... you win again :)

**Comment by alexgblr** [comment]
please merge. I created separate viv for that one.

**Review by mauricext4fs — APPROVED** [review]

**Pull request merged**